homeassistant/components/air_quality/__init__.py

```diff
@@ -19,7 +19,7 @@
 
 _LOGGER: Final = logging.getLogger(__name__)
 
-DOMAIN_DATA: HassKey[EntityComponent[AirQualityEntity]] = HassKey(DOMAIN)
+DATA: HassKey[EntityComponent[AirQualityEntity]] = HassKey(DOMAIN)
```

Comment:
Shouldn't we add a suffix that explains what data is stored? That is the existing pattern I think. Eg `DATA_COMPONENT`.